dev/com.ibm.jbatch.jsl.model/bnd.bnd

```diff
@@ -16,7 +16,6 @@ Bundle-SymbolicName: com.ibm.jbatch.jsl.model
 Bundle-Description:  Java Batch RI model
 
 # java batch is part of EE7 and therefore requires java 1.7
```

Comment:
Remove this comment.

Reply:
I have removed this comment and also updated all other bnd files that had that comment.